.pylintrc

```diff
@@ -428,7 +428,11 @@ disable=raw-checker-failed,
         suppressed-message,
         useless-suppression,
         deprecated-pragma,
-        use-symbolic-message-instead
+        use-symbolic-message-instead,
+        logging-fstring-interpolation,
+        dangerous-default-value,
+        too-many-arguments,
+        fixme
```

Comment:
My opinion on disabling warning like these is that, even if we do it, its better to keep it locally at a case by case basis. These warnings are good and adhering to them help us keep our code in a better quality. So its good not to disable them globally atleast

Reply:
I noticed the pattern in the repository and followed it. Most of the warnings have been disabled on a per-file basis, rather than globally in the pylintrc.

The four globally disabled warnings were added because I believe they are too opinionated and don't align with the coding style of this project. We can delve deeper into this topic if needed.

Please let me know if there's a specific warning you'd like to address

Let me know if there is a particular message you want

Reply:
@kavitharaju, Yes, there are 4 left over lint errors but I felt you are probably a better option to address them as I am not sure about the intention of the code. They are 2 lines in` test_chat_on_websocket.py` file:

`app/tests/test_chat_on_websocket.py:49:4: E1120: No value for argument 'vectordb' in function call (no-value-for-parameter)`